backend/.env.example

```
@@ -2,38 +2,41 @@
 # EMBEDDING_MODEL=BAAI/bge-small-en-v1.5
 # EMBEDDING_MAX_BATCH_SIZE=32
 # EMBEDDING_DEVICE=cpu
 
-# FalkorDB Configuration
-FALKORDB_HOST=localhost
-FALKORDB_PORT=6379
-CODEGRAPH_BACKEND_URL=http://localhost:5000
-CODEGRAPH_SECRET_TOKEN=DevRAI_CodeGraph_Secret
+#1. backend only
+BACKEND_URL=http://localhost:8000
+ 
```

dotenv-linter (4.0.0)

[warning] [TrailingWhitespace] Trailing whitespace detected

(TrailingWhitespace)